pam/integration-tests/cli_test.go

```diff
@@ -44,10 +40,10 @@ func TestCLIAuthenticate(t *testing.T) {
 		"Authenticate user with mfa":                                           {tape: "mfa_auth"},
 		"Authenticate user with form mode with button":                         {tape: "form_with_button"},
 		"Authenticate user with qr code":                                       {tape: "qr_code", pamUser: "user-integration-qr-code"},
-		"Authenticate user with qr code in a TTY":                              {tape: "qr_code", pamUser: "user-integration-qr-code-tty", termEnv: "linux"},
-		"Authenticate user with qr code in a TTY session":                      {tape: "qr_code", pamUser: "user-integration-qr-code-tty-session", termEnv: "xterm-256color", sessionEnv: "tty"},
-		"Authenticate user with qr code in screen":                             {tape: "qr_code", pamUser: "user-integration-qr-code-screen", termEnv: "screen"},
-		"Authenticate user with qr code after many regenerations":              {tape: "qr_code_quick_regenerate"},
+		"Authenticate user with qr code in a TTY":                              {tape: "qr_code", tapeSettings: []tapeSetting{{"Height", 650}}, pamUser: "user-integration-qr-code-tty", termEnv: "linux"},
```

Comment:
Maybe we can define the settings "keys" as consts in the helper file?

Reply:
You mean having a list of valid keys or define the defaults?

Reply:
AH... ok I guess I got what you mean.

Reply:
I'm using `const` values now, see if that's what you meant please :)